Template/Template/Controllers/ProfileViewController.swift

```diff
@@ -0,0 +1,13 @@
+// ProfileViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// контроллер экрана профиль
```

Comment:
```suggestion
/// Контроллер экрана профиль
```
с большой буквы описания начинаются